docs_espressif/zh_CN/debugproject.rst

```diff
@@ -1 +1,215 @@
-.. include:: ../en/debugproject.rst
\ No newline at end of file
+调试项目
+========
+
+:link_to_translation:`en:[English]`
+
+目录
+====
+
+.. contents::
+   :depth: 2
+   :local:
+
+
+启动调试会话
+------------
+
+在调试项目之前，请先指定设备的串口：
+
+1. 选择串口：
+
+- 点击 **查看** > **命令面板**。
+
+- 输入 **ESP-IDF：选择要使用的端口**，指定设备的串口。
+
+2. 确保 OpenOCD 配置文件正确
+
+- 点击 **查看** > **命令面板**。
+
+- 输入 **ESP-IDF：选择 OpenOCD 开发板配置**，选择扩展中 OpenOCD 服务器的配置文件。
+
+.. note::
+
+    * 请查看 `根据目标芯片配置 OpenOCD <https://docs.espressif.com/projects/esp-idf/zh_CN/latest/esp32/api-guides/jtag-debugging/tips-and-quirks.html#jtag-debugging-tip-openocd-configure-target>`_，为目标硬件选择开发板或配置。
+    * 请确保按照 `配置 JTAG 接口 <https://docs.espressif.com/projects/esp-idf/zh_CN/latest/esp32/api-guides/jtag-debugging/configure-ft2232h-jtag.html>`_ 中的说明来配置驱动程序。
+
+- （适用于 Linux 用户）在运行 OpenOCD 并启动调试会话前，请确保将 `OpenOCD udev 规则文件 <https://github.com/espressif/openocd-esp32/blob/master/contrib/60-openocd.rules>`_ 复制到 ``/etc/udev/rules.d`` 目录中。
+
+在菜单栏中点击 **运行** > **启动调试**，或者直接按 F5 启动调试。
+
+.. image:: ../../media/tutorials/debug/init_halted.png
```

Comment:
Is it possible to add a Chinese version of the UI in the screenshot? Or how about adding the English UI text in parentheses to help readers match it with the figure? If that’s too much work, just ignore this comment!

Reply:
Since the ``Run`` > ``Start Debugging`` command is part of VS Code rather than our extension, I suggest revising this sentence to "在顶部菜单栏中点击 ``运行`` > ``启动调试``". For other descriptions of specific screenshots, I will add the English UI text in parentheses if necessary. Thanks for your suggestion~